src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
```

Comment:
I don't think we need to do it for this PR. But at some point we need to settle on a convention and stick with it.

Whatever is decided, would you mind opening an issue for it?